devel/64tass/Portfile

```diff
@@ -0,0 +1,45 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           makefile 1.0
+
+name                64tass
+version             1.56.2625
+categories          devel
+maintainers         {patater.com:jaeden @Patater} \
+                    openmaintainer
+license             GPL-2+ LGPL-2.0+ LGPL-2.1+ MIT
+description         A multi pass optimizing macro assembler for the 65xx series of processors
+
+long_description \
+    64tass, sometimes also known as tass64, is a multi-pass optimizing macro \
+    assembler for the 65xx series of processors written in portable C. tass64 \
+    is syntax compatible with the independent, though similarly named, native \
+    Commodore 64 assembler TurboAssembler by Omicron.
+
+homepage            http://tass64.sourceforge.net/
+master_sites        sourceforge:tass64
+checksums           rmd160  6057e0d12541545a0966e82c9907d277ebf81d26 \
+                    sha256  c4e570c717c9500f3af61a3ad5d536f22415e2b29ed1eb09b1a955d310c9f3d3 \
+                    size    724673
+
+use_zip             yes
+distname            ${name}-${version}-src
+
+makefile.prefix_name prefix
+makefile.override-append PREFIX
+
+# Install docs and syntax files not installed by the Makefile
+post-destroot {
+    set docdir ${destroot}${prefix}/share/doc/${name}
+    xinstall -m 644 -W ${worksrcpath} \
+        NEWS \
+        README \
+        ${docdir}
```

Comment:
I haven't checked, but are the `LICENSE*` files installed through the Makefile? If not, you should probably add those as well to comply with the license.

Reply:
Yes, I've checked and they are. I'll mention this in the commit message.